src/reducers/project-title.js

```diff
@@ -31,6 +21,5 @@ const setProjectTitle = title => ({
 export {
     reducer as default,
     initialState as projectTitleInitialState,
-    defaultProjectTitleMessages,
```

Comment:
This is going to conflict with #3391 — would you coordinate with @benjiwheeler on how this should work?

Reply:
@rschamp we agreed that @benjiwheeler will land #3391, and then I'll fix up conflicts. I'll move the `defaultProjectTitle` message into `sharedMessages` in gui. Then it can be used in multiple places.

Reply:
Thanks, this is cleaner!